app/models/minion.rb

```diff
@@ -10,6 +10,13 @@ class NotEnoughMinions < StandardError; end
   # Raised when we fail to assign a role on a minion
   class CouldNotAssignRole < StandardError; end
 
+  # Human readable names for highstate attribute values
+  HIGHSTATES = {
+    pending:  nil,
+    complete: true,
+    failed:   false
+  }.freeze
```

Comment:
What do you mean? I was thinking of making this column and integer in the database and keep a mapping hash here (like in the code above).